kuma/javascript/src/active-banner.jsx

```diff
@@ -191,7 +206,11 @@ export default function ActiveBanner() {
     }
 
     for (const id in userData.waffle.flags) {
-        if (!userData.waffle.flags[id] || isEmbargoed(id)) {
+        if (
+            !userData.waffle.flags[id] ||
+            isEmbargoed(id) ||
+            notForThisUser(id, userData)
```

Comment:
Since that function only does one thing. I could move its logic to somewhere here. Something like this:

```diff
if (!userData.waffle.flags[id] || isEmbargoed(id)) {
    continue;
}
+// Code comment...
+if (banner === SUBSCRIPTION_ID && userData.isSubscriber)  {
+   continue;
+}
```

would you prefer that?

Reply:
I think it's simple enough that I'd actually prefer that, yup!